Blog/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import path
 
 from Blog.Feeds.blogpost_feed import LatestPostsFeed
+from DiyBlog.settings import DEBUG
```

Comment:
You should read the settings from Django instead: https://docs.djangoproject.com/en/2.0/topics/settings/#using-settings-in-python-code

Django sets some default values for settings you have not defined in your app. This way you could remove `DEBUG` from you settings without getting an attribute error.